exonum-java-binding-core/src/test/java/com/exonum/binding/transport/VertxServerIntegrationTest.java

```diff
@@ -164,4 +166,15 @@ private void blockingStop() throws InterruptedException, ExecutionException, Tim
     int stopTimeout = 2;
     f.get(stopTimeout, TimeUnit.SECONDS);
   }
+
+  /**
+   * Returns random available local port.
+   */
+  private int findFreePort() throws IOException {
+    try (ServerSocket socket = new ServerSocket(0)) {
+      socket.setReuseAddress(true);
```

Comment:
Is it needed (i.e., isn't the socket freed after this method returns)? If I understand correctly, it won't affect a new socket that will be created by the server unless that server uses the same option.

Reply:
yes, this statement can be removed.